openfisca_us/parameters/gov/states/md/tax/income/married.yaml

```diff
@@ -0,0 +1,46 @@
+description: MD State income tax rate for single filers
+metadata:
+  type: single_amount
```

Comment:
Should this be `marginal_rate` instead? Changing amounts to rates below as well per https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#creating-scales